internal/cri/config/config.go

```diff
@@ -102,6 +102,8 @@ type Runtime struct {
 	// to the runtime spec when the container when PrivilegedWithoutHostDevices is already enabled. Requires
 	// PrivilegedWithoutHostDevices to be enabled. Defaults to false.
 	PrivilegedWithoutHostDevicesAllDevicesAllowed bool `toml:"privileged_without_host_devices_all_devices_allowed" json:"privileged_without_host_devices_all_devices_allowed"`
+	// CgroupWritable enables writable cgroups in non-privileged containers
+	CgroupWritable bool `toml:"cgroup_writable" json:"cgroupWritable"`
```

Comment:
This should be rejected on cgroup v1

Reply:
Thanks @AkihiroSuda for the review! I have added a check in [ValidateRuntimeConfig](https://github.com/containerd/containerd/blob/0aef239a9e506655952ce117ab489c351cc1f924/internal/cri/config/config.go#L533)